src/lib/utilities/get-single-attribute-for-event.ts

```diff
@@ -188,3 +201,38 @@ export const getSingleAttributeForEvent = (
 
   return getSummaryAttribute(event);
 };
+
+export const checkForChildWorkflowExecutionAndAddRunIdAttribute = ({
```

Comment:
A few things to consider...
1) The util `checkForChildWorkflowExecutionAndAddRunIdAttribute` mutates the `currentEvent`. This is the same behavior as before, but now I'm wondering if we want to make a copy and return the `currentEvent` from the util instead 🤔 
2) There are instances where the `StartChildWorkflowExecutionInitiated` event is not immediately followed by a `ChildWorkflowExecutionStarted` event, which means that this logic doesn't work and the `Workflow Id` will not have a link. The logic gets increasingly complex if I try to iterate through events to account for this, but I think it might be possible. Maybe by looking for a `ChildWorkflowExecutionStarted`  event with a matching workflow id? Curious if you think it's worth investigating...

Reply:
**Update:** We have decided not to add the child execution link to the `Workflow ID` on the `StartChildWorkflowExecutionInitiated` event. 

The `StartChildWorkflowExecutionInitiated` event does not have the run ID needed to create the link. The run ID is created and added to the workflow execution in the following `ChildWorkflowExecutionStarted` event. 

Rather than loop through following events (which may not all be fetched in the UI) to find the `ChildWorkflowExecutionStarted` event with the run ID needed for the link on the `StartChildWorkflowExecutionInitiated` event, we have determined it just makes sense to only have a link when the run ID exists for that event.